### ISSUE TITLE
Include Annotation name in Annotation report

### ISSUE THREAD

**Issue by torquil** [Slicer/Slicer#4902]
In the Annotations module:

The Annotation report contains only type, value and text, not the annotation name. So I suggest that the Annotation name is also included in the annotation report.

**Comment by jamesobutler** [comment]
I believe the annotations module is scheduled to be replaced fully by the markups module.  @lassoan Do you have plans to carry over the "Display a report of highlighted annotations" to the markups module? Or just require copy/paste of the tables for a given markup node and its points?

**Comment by lassoan** [comment]
Yes, we are going to remove Annotations module (when all the features get implemented in Markups module), so we won't implement any enhancements, so I'll close this issue.

It should not be hard to generate an HTML report from markup nodes, but a good report tends to be very application-specific (what information you want to show and how, in what layout). Can you describe what would you like to achieve, maybe create a mockup report in Word?
